src/pages/Categories.js

```diff
@@ -0,0 +1,7 @@
+import React from 'react';
+
+const Categories = () => (
+  <div>Categories Page Coming Soon</div>
```

Comment:
- [ ] @Stephen-Adom great work on your pages however the categories page should have a button that checks for a status. Kindly add a button
![image](https://user-images.githubusercontent.com/61408860/228891603-2ed1f358-ebcc-494b-81ee-7c2ec5b72613.png)